packages/grafana-data/src/types/config.ts

```diff
@@ -220,8 +220,11 @@ export interface GrafanaConfig {
 }
 
 export interface AuthSettings {
+  // deprecation notice
+  // in favor of setting one for each specific auth integration
```

Comment:
How are deprecation notices added for settings? This is an internal file and is not very visible, so I think we'll need to add more than this for customers to be aware of the change. I'd prefer creating a new issue to track the work so that we make sure to do it properly.

Reply:
the idea was to add a comment around it. was not meant as a feature, will instead make a new issue where we add the feature 👍